python/fbprophet/forecaster.py

```diff
@@ -123,6 +131,10 @@ def __init__(
         self.weekly_seasonality = weekly_seasonality
         self.daily_seasonality = daily_seasonality
 
+        if (holidays is not None) and (append_holidays is not None):
+            raise ValueError("User cannot set up both holidays and append_holidays!")
```

Comment:
Does it seem reasonable to allow users to specify both `holidays` and `append_holidays` (or whatever its final name will be) and then just append the `append_holidays` to any holidays they provided. That would make a really easy way for people to start from the hdays list and add minor customizations that they need for their application.